thunder/tests/opinfos.py

```diff
@@ -1090,6 +1091,14 @@ def _abs_torch(x: torch.Tensor | Number):
 )
 elementwise_unary_ops.append(floor_opinfo)
 
+is_complex_opinfo = OpInfo(
```

Comment:
`is_complex` is more tensor property like `is_cuda`, so let's put its OpInfo with the other tensor properties:

https://github.com/khushi-411/lightning-thunder/blob/05524ae8b828640587f6a67c72cd0c78671df649/thunder/tests/opinfos.py#L548